### PR TITLE
Site config update

### PR DESCRIPTION
**Pull request by PetrDlouhy**
I have added few parameters (`blog_about_url`, `small_logo`) to the site config, parametrized Twitter feed by `twitter_link` and removed unused `header_html` and `footer_links_html` parameters (I think, it is better to parametrize the urls, because the link texts can be translated then).

### CODE REVIEW

**Comment by mvl22** [comment]
Great, thanks - all sounds sensible. Nikolai I'm sure will review shortly.

File: app/views/home/_twitter.html.haml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<a class="twitter-timeline" data-lang="en" data-height="1023" data-dnt="true" href="https://twitter.com/Cyclescape">Tweets by Cyclescape</a>
+%a{href: @site_config.twitter_link, class: "twitter-timeline", title: "Tweets by Cyclescape", "data-height" => "1023", "data-dnt" => "true"}
```
Comment (nikolai-b) [inline app/views/home/_twitter.html.haml]: You don't want to make it 
```ruby
%a{href: @site_config.twitter_link, class: "twitter-timeline", title: t(".title"), "data-height" => "1023", "data-dnt" => "true"}
```
and add the twitter tile to the language config?

**Comment by nikolai-b** [comment]
@PetrDlouhy Thanks, I'm was going to have the header and footer html but you're right, it is much better to translate individual chunks of text separably than have a block of html.

**Comment by PetrDlouhy** [comment]
@nikolai-b I have reworked the Twitter title, although it doesn't change much, because it is overridden by JavaScript.

**Comment by nikolai-b** [comment]
It is overwritten by JavaScript? Oops, sorry then.

**Pull request merged**